client/app/vrack/vrack.html

```diff
@@ -56,6 +56,14 @@ <h6 class="cui-page-header__subtitle" data-ng-bind="VrackCtrl.serviceName" data-
     </div>
 
     <div class="cui-page__content">
+        <section>
```

Comment:
Something very similar to your feature has been implemented here: https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/cloud/project/compute/cloud-project-compute.controller.js#L80

See line https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/cloud/project/compute/cloud-project-compute.controller.js#L92 on how it uses CloudMessage, it's very easy :)

As in my previous comment, if you could try to extract the 3 (vrack, pci compute and VPS) implementations in using one service, it would be great.

If you believe it is out of scope, you can simply replicate what is done in pci-compute. But we should really try to follow the DRY principle here (Don't Repeat Yourself).